electron/legendary_utils/library.ts

```diff
@@ -62,6 +62,8 @@ export async function getLegendaryConfig(file: string) {
             dlcItemList,
             customAttributes: { CloudSaveFolder, FolderName },
           } = metadata
+          
+          const {catalog_item_id, namespace} = asset_info
```

Comment:
I do not see `catalog_item_id` being used anywhere, what is the use / reason of it being added?